steps/apply.ts

```diff
@@ -1,4 +1,4 @@
-import { Page } from '@playwright/test'
+import { Page, expect } from '@playwright/test'
```

Comment:
Is this import required?

Reply:
ah nice spot, this is for the next commit